packages/firebase_remote_config/firebase_remote_config_platform_interface/lib/src/remote_config_settings.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:meta/meta.dart';
+
+/// Defines the options for the corresponding Remote Config instance.
+class RemoteConfigSettings {
+  /// Constructs an instance of [RemoteConfigSettings] with given [fetchTimeout]
+  /// and [minimumFetchInterval].
+  RemoteConfigSettings({
+    @required this.fetchTimeout,
+    @required this.minimumFetchInterval,
```

Comment:
Those parameters have a default value as stated in the comment below (1min for `fetchTimeout`, 12h for `minimumFetchInterval`), so they shouldn't be `@required`, should they?

Reply:
We could add the defaults here in which case they would not (could not) be required. However I'd hope to avoid devs resetting a RemoteConfigSettings property to default after it was previously changed without explicitly setting it. example:`RemoteConfigSettings(fetchTimeout: Duration(seconds: 10))` you would also be resetting the `minimumFetchInterval` to 1 hour even if it was previously set to 0 seconds.

Reply:
What about something like "fetch(force: true)"?

Reply:
We are hoping to stay as close to the native API as possible so for now we should avoid adding features/parameters that are not available on the native side if possible.